packages/cli/src/bases/Package.ts

```diff
@@ -38,6 +38,11 @@ export namespace Package {
             return `yarn add ${pkg}`;
           case "pnpm":
             return `pnpm add ${pkg}`;
+          default:
```

Comment:
We're already handling this at the TypeScript level.
Is there a specific reason for handling it separately?

Reply:
If there is uncertainty in the information, this should be confirmed by the individual functions. ‘I'm processing that in a totally separate and remote place’ can't be a reason.

Frankly, I think the only benefit of adding this code would be increased safety.

Not only in this code, but in the entire code base, it is also difficult to write tests because it is uncertain whether individual functions are fulfilling their duties.

At any rate, it is not mutually beneficial to both parties to pay the cost of explaining this. Let's merge them anyway.


Translated with www.DeepL.com/Translator (free version)

Reply:
If there is uncertainty in the information, this should be confirmed by the individual functions. ‘I'm processing that in a totally separate and remote place’ can't be a reason.

Frankly, I think the only benefit of adding this code would be increased safety.

Not only in this code, but in the entire code base, it is also difficult to write tests because it is uncertain whether individual functions are fulfilling their duties.

At any rate, it is not mutually beneficial to both parties to pay the cost of explaining this. Let's merge them anyway.

Reply:
The argument values are limited.
Still, if you think of it as an uncertainty, it can be
The syntax doesn't seem to be a problem, so we'll hit Resolve.